src/Network/Network/PrivateEndpoint/NewAzurePrivateEndpointCommand.cs

```diff
@@ -91,6 +92,9 @@ public class NewAzurePrivateEndpoint : PrivateEndpointBaseCmdlet
             HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Application Security Groups")]
+        public PSApplicationSecurityGroup[] ApplicationSecurityGroups { get; set; }
```

Comment:
Please use a singular name

Reply:
@vidai-msft forgot to mention. Please update the help doc for the cmdlet as you add a parameter